eclipsePlugin/doc/building_spotbugs_plugin.txt

```diff
@@ -41,9 +41,10 @@ Import the projects
 8) Choose [File | Import General | Existing Projects into Workspace | Next]. The Import project import wizard opens.
 9) In the wizard, proceed as follows:
    a) Choose [Select root directory].
-   b) Enter ~/git/spotbugs. Click [Browse]
-   c) Click [Finish].
-   g) Now "spotbugs" project will be compiled by Eclipse and should not have any errors,
+   b) Enter ~/git/spotbugs. Click [Browse]. Make sure the checkbox "Search for nested projects" is selected.
```

Comment:
This is @Bananeweizen's commit, not mine.  I thought it looked reasonable, but we can of course change it to something better :)